### PR TITLE
Pre-Publish Checklist: Update text strings for links, images, and videos.

### PR DESCRIPTION
**Pull request by carloskelly13**
## Summary

- Updates the text to match what was found for videos, images, and links.

## User-facing changes

new text strings for missing alternative text for videos and images and too many links per page.

## Testing Instructions

1. Place more than 3 links on a page to trigger the error
2. note the text of the error states "Avoid having more than 1 link per page"
3. Place an image or video with no alternative text

---

#5381 #5382

### CODE REVIEW

**Review by BrittanyIRL — APPROVED** [review]
LGTM, text is updated: 
Screen Shot 2020-11-24 at 9 20 31 AM

**Review by maxyinger — APPROVED** [review]

**Comment by csossi** [comment]
Not seeing any messages re: missing assistive text for either image or video

**Review by zachhale — APPROVED** [review]

File: assets/src/edit-story/app/prepublish/constants.js [hunk 1]
```diff
@@ -186,25 +186,17 @@ export const MESSAGES = {
...
     MISSING_VIDEO_ALT_TEXT: {
-      MAIN_TEXT: __('Video missing alt text', 'web-stories'),
-      HELPER_TEXT: sprintf(
-        /* translators: %s: alt. */
-        __(
-          'Add meaningful “%s” text to videos to optimize accessibility and indexability.',
-          'web-stories'
-        ),
-        'alt'
+      MAIN_TEXT: __('Video missing assistive text', 'web-stories'),
```
Comment (dreamofabear) [inline assets/src/edit-story/app/prepublish/constants.js]: "Video missing description"

See #5332.

**Comment by csossi** [comment]
Still not seeing message

Deleted assistive text here:


Checklist doesn't show message:


Looks like updating assistive text in the design panel doesn't impart the "three-dot" menu for the video/image:

File: assets/src/edit-story/app/prepublish/constants.js (hunk 1 above)
Comment (carloskelly13) [reply assets/src/edit-story/app/prepublish/constants.js]: Ah, we are changing that again.

**Comment by carloskelly13** [comment]
@csossi I see this working; so I'm going to toss this back. Let me know if there are any issues.

Screen Shot 2020-12-01 at 12 04 08 PM
Screen Shot 2020-12-01 at 12 03 54 PM

**Comment by swissspidy** [comment]
@csossi Are you testing the correct PR? If you test #5417 (this PR here) you should still see 2 fields for video description. But in your screenshot there is only 1 field, which means you are probably testing #5332

**Comment by csossi** [comment]
you're right, @swissspidy - I WAS testing 5332! Thanks for pointing that out

**Comment by csossi** [comment]
Verified in QA

**Pull request merged**